pages/index.tsx

```diff
@@ -235,7 +243,8 @@ const sorts = {
   [SortOptions.RecentlyAdded]: [descend(prop('createdAt')), descend(currentListingPrice)],
   [SortOptions.Expensive]: [descend(prop('highestBid')), descend(currentListingPrice)],
   [SortOptions.Cheapest]: [ascend(currentListingPrice), ascend(prop('endsAt'))],
-  [SortOptions.BidCount]: [descend(prop('totalUncancelledBids'))]
+  [SortOptions.BidCount]: [descend(prop('totalUncancelledBids'))],
+  [SortOptions.Trending]: [descend(prop('totalUncancelledBids')), ascend(prop('endsAt'))]
```

Comment:
👍

Reply:
We can drop bid count right? not on .com and trending is pretty much the same thing but better.